### DISCUSSION TITLE
git 2.0 dependency, on Windows?

### DISCUSSION THREAD

**Discussion by leeand00** [jupyterlab/jupyterlab-git#1047] [Q&A] [answered]
I know I should have Juypter 3+ and git 2.0, but what installer should I use to get support for git on Windows 10?

![image](https://user-images.githubusercontent.com/83778/136121968-417bbf05-7aa3-4648-82e6-757642dec164.png)

I have just installed `git version 2.33.0.windows.2`, but when I restart `jupyterlab` with the `jupyterlab-git` extension.

**Comment by fcollonval** [comment, marked as answer]
Hey @leeand00, could you try to run a git command on the Windows Command Line?

For that you have to:
1. Press shortcut `Win + R`
2. Type `cmd` and press `Enter`
3. Type in the opened command window, `git --version`

If you get an error `not found`, this means Windows cannot figure out where `git` is installed. You will need to add the path to git to your `%PATH%` environment variable; see [that thread](https://stackoverflow.com/questions/4492979/git-is-not-recognized-as-an-internal-or-external-command)

**Reply by leeand00** [reply]
I'm sure it's not in my %PATH%; but since it installs with git bash, I didn't think it was possible to do that, I figured it ran in bash and not cmd.exe; I guess I just have to search for `git.exe`?

**Reply by fcollonval** [reply]
Indeed you need to search for `git.exe`. The stackoverflow discussion I linked in the previous post mentioned some paths.

> The reason is that the extension runs the git commands as subprocesses. So `git` executable must be in your `%PATH%`.

**Reply by leeand00** [reply]
Okay I ended up downloading `Git-2.33.0.2-64-bit.exe` from the git scm website and adding `C:\Program Files\Git\bin\` to my `%path%`; now it works, thank you.
